### PR TITLE
Upgrade pnpm to version 7

### PR DESCRIPTION
**Pull request by scopsy**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Upgrade for pnpm v7
- **What is the current behavior?** (You can also link to an open issue here)
- Currently only support pnpm v6
- **What is the new behavior (if this is a feature change)?**

- **Other information**:

### CODE REVIEW

File: apps/api/package.json [hunk 1]
```diff
@@ -80,6 +80,7 @@
     "newrelic": "^7.4.0",
     "passport": "^0.4.1",
     "passport-github2": "^0.1.12",
+    "passport-oauth2": "1.6.1",
```
Comment (scopsy) [inline apps/api/package.json:83]: Had to add peer dependencies manually since last pnpm major change.

File: apps/api/Dockerfile [hunk 2]
```diff
@@ -2,10 +2,11 @@ FROM node:16.13.1
 
 WORKDIR /usr/src/app
 
-RUN npm i pnpm@6.32.11 -g --loglevel notice --force
-RUN pnpm i pm2 -g
+RUN npm install -g pnpm@7.3.0 -g --loglevel notice
```
Comment (BiswaViraj) [inline apps/api/Dockerfile]:
```suggestion
RUN npm install pnpm@7.3.0 -g --loglevel notice
```
Double `-g`

File: apps/web/Dockerfile [hunk 3]
```diff
@@ -2,8 +2,9 @@ FROM node:16.13.1
 
 WORKDIR /usr/src/app
 
-RUN npm i pnpm@6.32.11 -g --loglevel notice --force
+RUN npm install -g pnpm@7.3.0 -g --loglevel notice
```
Comment (BiswaViraj) [inline apps/web/Dockerfile]:
```suggestion
RUN npm install pnpm@7.3.0 -g --loglevel notice
```

File: apps/widget/Dockerfile [hunk 4]
```diff
@@ -2,8 +2,9 @@ FROM node:16.13.1
 
 WORKDIR /usr/src/app
 
-RUN npm i pnpm@6.32.11 -g --loglevel notice --force
+RUN npm install -g pnpm@7.3.0 -g --loglevel notice
```
Comment (BiswaViraj) [inline apps/widget/Dockerfile]:
```suggestion
RUN npm install pnpm@7.3.0 -g --loglevel notice
```

File: apps/api/Dockerfile (hunk 2 above)
Comment (djabarovgeorge) [inline apps/api/Dockerfile:5]: Is the double global flag needed?

Comment (scopsy) [reply apps/api/Dockerfile:5]: Nope hehe good catch

File: apps/ws/Dockerfile [hunk 5]
```diff
@@ -2,9 +2,10 @@ FROM node:16.13.1
 
 WORKDIR /usr/src/app
 
-RUN npm i pnpm@6.32.11 -g --loglevel notice --force
-RUN pnpm i pm2 -g
+RUN npm install -g pnpm@7.3.0 -g --loglevel notice
```
Comment (BiswaViraj) [inline apps/ws/Dockerfile]:
```suggestion
RUN npm install pnpm@7.3.0 -g --loglevel notice
```

File: libs/embed/Dockerfile [hunk 6]
```diff
@@ -2,8 +2,9 @@ FROM node:16.13.1
 
 WORKDIR /usr/src/app
 
-RUN npm i pnpm@6.32.11 -g --loglevel notice --force
+RUN npm install -g pnpm@7.3.0 -g --loglevel notice --force
```
Comment (BiswaViraj) [inline libs/embed/Dockerfile]:
```suggestion
RUN npm install pnpm@7.3.0 -g --loglevel notice --force
```

**Comment by BiswaViraj** [comment]
Ah, nvm I was slow while submitting the review 😅 
Anyway, great work

File: apps/ws/package.json [hunk 7]
```diff
@@ -21,6 +21,7 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "jsonwebtoken": "8.5.1",
```
[the remark this answers is not in the document]
Comment (scopsy) [reply apps/ws/package.json:24]: Used in the UserDecorator to checkout the payload of the jwt token

**Comment by scopsy** [comment]
Thanks @BiswaViraj @djabarovgeorge and @davidsoderberg  for the review!

**Pull request merged**